notebooks/ICESat-2_Cloud_Access/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -53,7 +53,7 @@
     "### **Example of end product** \n",
```

Comment:
<p>Suggest "To display the rendered metadata, including download link, granule size and two images, use <code>display</code>. In the example below, only the first 4 results are shown.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Have made these updates to the text